### DISCUSSION TITLE
Video is downloading with wronh language audio track

### DISCUSSION THREAD

**Discussion by limacub777** [aclap-dev/video-downloadhelper#1027] [Help and Q&A]
### Your question

https://watch.thechosen.tv/video/184683594325?language=en 
is downloading with Arabic audio track even when english is selected. Please assist.

I need it in English.

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: en-US

**Comment by GrampaWildWilly** [comment]
#956
#985

**Reply by limacub777** [reply]
Thanks

**Reply by GrampaWildWilly** [reply]
Please edit your discussion title to correct the misspelling of `wrong`.  This is a service to anybody searching the forum.  You can edit your own posts, including the title of the thread, by clicking the `...` above & to the right of any of your posts.

**Comment by limacub777** [comment]
Really? All this because of a typo? JeezDo you have anything more productive to do than checking spelling?On Jul 19, 2024, at 8:03 PM, GrampaWildWilly ***@***.***> wrote:﻿
Please edit your discussion title to correct the misspelling of wrong.  This is a service to anybody searching the forum.  You can edit your own posts, including the title of the thread, by clicking the ... above & to the right of any of your posts.

**Reply by GrampaWildWilly** [reply]
I wasn't specifically looking for it.  I just happened to notice it.  If you think about it, it actually reflects badly on you.  With the effort it took to speak condescendingly to me, you could have fixed it.
